doc/api/http.md

```diff
@@ -48,25 +48,25 @@ list like the following:
 added: v0.3.4
 -->
 
-The HTTP Agent is used for pooling sockets used in HTTP client
+The HTTP `Agent` is used for pooling sockets used in HTTP client
 requests.
 
-The HTTP Agent also defaults client requests to using
+The HTTP `Agent` also defaults client requests to using
 `Connection: keep-alive`. If no pending HTTP requests are waiting on a
 socket to become free the socket is closed. This means that Node.js's
-pool has the benefit of keep-alive when under load but still does not
-require developers to manually close the HTTP clients using
-KeepAlive.
-
-If you opt into using HTTP KeepAlive, you can create an Agent object
-with that flag set to `true`.  (See the [constructor options][].)
-Then, the Agent will keep unused sockets in a pool for later use.  They
-will be explicitly marked so as to not keep the Node.js process running.
-However, it is still a good idea to explicitly [`destroy()`][] KeepAlive
-agents when they are no longer in use, so that the Sockets will be shut
-down.
-
-Sockets are removed from the agent's pool when the socket emits either
+pool has the benefit of HTTP Keep-Alive when under load but still does not
+require developers to manually close HTTP clients when using
+this option.
+
+If you opt into using HTTP Keep-Alive, you can create an `Agent` instance,
+providing `{ keepAlive: true }` among the constructor options. (See the
```

Comment:
How about just link 'constructor options' here instead of having the extra '(See the constructor options.)'